content/docs/gke/custom-domain.md

```diff
@@ -4,54 +4,67 @@ description = "Using a custom domain with Kubeflow on GKE"
 weight = 30
 +++
 
-{{% needs-update %}}
-
-This guide describes how to use a custom domain with Kubeflow on Google Cloud 
+This guide describes how to use a custom domain with Kubeflow on Google Cloud
 Platform (GCP).
 
 ## Before you start
 
 This guide assumes you have already set up Kubeflow on GCP. If you haven't done
-so, follow the guide to 
+so, follow the guide to
 [getting started with Kubeflow on GCP](/docs/gke/deploy/).
 
 ## Using your own domain
 
-If you want to use your own domain instead of **${KF_NAME}.endpoints.${PROJECT}.cloud.goog**, follow these instructions:
+If you want to use your own domain instead of **${KF_NAME}.endpoints.${PROJECT}.cloud.goog**, follow these instructions after running `kfctl build`:
+
+1. Replace `hostname` in `kustomize/iap-ingress/base/params.env` with your own domain.
+
+   ```
+   ...
+   hostname=<enter your domain here>
+   ingressName=envoy-ingress
+   ipName=kf-test-ip
+   oauthSecretName=kubeflow-oauth
+   ...
+   ```
+
+2. Apply the changes.
+
+   Using `kfctl`.
```

Comment:
The list formatting is broken from this point onwards on the rendered page. See preview: https://deploy-preview-1650--competent-brattain-de2d6d.netlify.com/docs/gke/custom-domain/

I think indenting by 2 more characters should fix it. Then you can change the list indicators back to `1.` instead of using `2.`, `3.`, and so on.